src/components/Datepicker.vue

```diff
@@ -740,7 +750,11 @@ export default {
     },
     setPageDate (date) {
       if (!date) {
-        date = new Date()
+        if (this.openDate) {
+          date = new Date(this.openDate)
```

Comment:
This line is missed in the tests.
https://coveralls.io/builds/14019344/source?filename=src%2Fcomponents%2FDatepicker.vue#L754